plugin/src/main/java/com/spotify/plugin/dockerfile/AbstractDockerMojo.java

```diff
@@ -192,6 +192,12 @@ public String getFileName() {
   private JarArchiver jarArchiver;
 
   /**
+   * Allows enabling/disabling gcr credentials
+   */
+  @Parameter(defaultValue = "true", property = "dockerfile.googleregistry")
```

Comment:
I think a more Maven idiomatic name for this parameter and field name would be something like "googleContainerRegistryEnabled". The "docker" prefix is redundant and I think a Boolean flag should end in "Enabled".